sound/soc/sof/ipc4-topology.c

```diff
@@ -721,8 +721,13 @@ static void sof_ipc4_unprepare_copier_module(struct snd_sof_widget *swidget)
 	pipeline = pipe_widget->private;
 	pipeline->mem_usage = 0;
 
-	if (WIDGET_IS_AIF(swidget->id))
+	if (WIDGET_IS_AIF(swidget->id)) {
 		ipc4_copier = swidget->private;
+	} else {
```

Comment:
maybe else if WIDGET_IS_DAI() here? we might have module-to-module copiers later on